### PR TITLE
Refactor supplier-frontend forms, part 1

### PR DESCRIPTION
**Pull request by lfdebrux**
This is part of the work for [my ticket](https://trello.com/c/AQVm3WXZ) to refactor the forms in supplier-frontend.

As this app has a very large number of forms, at @katstevens's suggestion I have split the work into two PRs part 1 and part 2 (to follow).

This PR deals with all the forms in the `app/main/forms/frameworks.py` and `app/main/forms/auth_forms.py` files. These are fairly straightforward forms which have mainly just been cleaned up using some of the new features from dmutils.

Notice I had to bring in a new version of dmutils to get some bugfixes.

### CODE REVIEW

File: app/templates/frameworks/contract_variation.html [hunk 1]
```diff
@@ -28,16 +28,16 @@
     <div class="grid-row">
       <div class="column-one-whole">
         {% if errors %}
-         {% with 
+         {% with
             message = "You must <a href=\"#{}\">accept the changes</a> to continue.".format(errors.accept_changes.input_name)|safe, type = "destructive" %}
```
Comment (katstevens) [inline app/templates/frameworks/contract_variation.html]: Double quotes inside double quotes here (my editor's gone red!)

File: app/main/forms/frameworks.py [hunk 2]
```diff
@@ -44,22 +51,25 @@ class ReuseDeclarationForm(FlaskForm):
...
     )
     old_framework_slug = HiddenField()
 
 
 class OneServiceLimitCopyServiceForm(FlaskForm):
 
+    copy_service = DMBooleanField(
+        "Do you want to reuse your previous {lot_name} service?",
+        question_advice="You still have to review your service and answer any new questions.",
+        false_values={"False", "false", ""},
```
[the remark this answers is not in the document]
Comment (lfdebrux) [reply app/main/forms/frameworks.py]: Documentation indicates a tuple, I'll make it consistent in our code

File: app/templates/frameworks/contract_variation.html (hunk 1 above)
Comment (lfdebrux) [reply app/templates/frameworks/contract_variation.html]: Yikes! I think the backslash should handle this? I don't think I actually touched that line...

**Comment by lfdebrux** [comment]
@katstevens I wish I could be totally confident the functional tests would catch everything...

**Pull request merged**